cmd/ausoceantv/src/shared/tailwind.element.ts

```diff
@@ -0,0 +1,14 @@
+// See Also: https://github.com/butopen/web-components-tailwind-starter-kit
+
+import {LitElement, unsafeCSS} from "lit";
+
+import style from "./tailwind.global.css?inline";
+
+const tailwindElement = unsafeCSS(style);
+
+export const TailwindElement = (style) =>
```

Comment:
It would be good if TailwindElement took style as an optional argument, because for most of the time we would be able to do all the styling using tailwind and not need additional css written.

Reply:
Good point. Done.